custom_components/volkswagen_we_connect_id/config_flow.py

```diff
@@ -85,7 +84,3 @@ async def async_step_user(
 
 class CannotConnect(HomeAssistantError):
     """Error to indicate we cannot connect."""
-
-
-class InvalidAuth(HomeAssistantError):
-    """Error to indicate there is invalid auth."""
```

Comment:
This exception class was defined but not raised anywhere, and therefore not capable of being caught through `except InvalidAuth`. I assume it was connected to that comment earlier in the file:
```python
# TODO: ADD Validation on credentials
```